presidio-cli/tests/test_cli.py

```diff
@@ -0,0 +1,99 @@
+import os
+import pytest
+from io import StringIO
+from presidio_cli import cli
+
+
+@pytest.fixture()
+def problems(mocker):
+    problem1 = mocker.Mock(
+        line=1,
+        column=7,
+        score=1.0,
+        type="PERSON",
+        explanation=None,
+        recognizer_result={},
+    )
+    problem2 = mocker.Mock(
+        line=2,
+        column=17,
+        score=0.85,
+        type="PERSON",
+        explanation="some example",
+        recognizer_result={},
+    )
+    return [problem1, problem2]
+
+
+def test_find_files_recursively(temp_workspace, config):
+    assert sorted(cli.find_files_recursively([temp_workspace], config)) == [
```

Comment:
This test is failing on Windows. 
The comparison of paths fails because paths on Windows look differently (e.g. `sub\\directory.txt\\empty.txt`). In addition, the `non-ascii/éçäγλνπ¥/utf-8` file fails the `is_text_file` not sure why.

For the first problem, a quick solution would be to add the folder names as inputs to `os.path.join`:
```python
    assert sorted(cli.find_files_recursively([temp_workspace], config)) == [
        os.path.join(temp_workspace, "dos.yml"),
        os.path.join(temp_workspace, "empty.txt"),
        os.path.join(temp_workspace, "errorfile"),
        os.path.join(temp_workspace, "non-ascii", "éçäγλνπ¥", "utf-8"),
        os.path.join(temp_workspace, *["s"]*15, "file"),
        os.path.join(temp_workspace, "sub", "directory.txt", "empty.txt"),
    ]
```

For the second issue, I'm getting this error: `UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 40: character maps to <undefined>`. The character in position 40 is `お`. Perhaps `encoding` is needed?

If I change the `config.is_text_file`'s logic to except `encoding="utf-8"` then those tests pass correctly. I'm not sure honestly if that's a right call to make. Happy to hear your view.

Reply:
Thanks for the suggestion. I corrected that. I added `encoding="utf-8"` to `open` calls in both `config.py` and `cli.py`. It seems to me that the reason for the failure could be a different default encoding used by Python on Windows.